airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/pipeline.py

```diff
@@ -190,6 +191,20 @@ def prepare_container_for_poe_tasks(
     # Set working directory to the poetry package directory
     container = container.with_workdir(f"/airbyte/{poetry_package_path}")
 
+    # If a package from `airbyte-platform-internal` is required, modify the entry in pyproject.toml to use https instead of ssh, when run in Github Actions
+    if is_ci:
+        container = container.with_exec(
+            ["sed", "-i", "-E", r"s,git@github\.com:airbytehq/airbyte-platform-internal,https://github.com/airbytehq/airbyte-platform-internal.git,", "pyproject.toml"]
```

Comment:
Can we make the URL `https://` in the checked in pyproject.toml.

Reply:
If we did that `poetry install` wouldn't work for regression tests locally (whether running with `pytest` or `airbyte-ci`) unless the user has a PAT in their environment.